### DISCUSSION TITLE
Add option to disable '/etc/init.d/S97webrtc'

### DISCUSSION THREAD

**Discussion by ClassicOldSong** [Guilouz/Creality-Helper-Script-Wiki#502] [Features Request]
With this service left on, the printer will record time lapse anyways, and upload to Creality Cloud in the background. Disable it could prevent the behavior.

Also, mdns could also be disabled since we won't need the discovery service provided by it if we're not using Creality Print

**Comment by kisslorand** [comment]
Thanks for the tips, I just disabled them in my K1.

**Reply by jattie-ire** [reply]
Can you share the details of your method to disable these services please?

**Comment by kisslorand** [comment]
I deleted from "/etc/init.d/" the following "S97webrtc" and "S99mdns".
Also in "/usr/bin/" I renamed "mdns" to "mdns.disabled" and "webrtc" to "webrtc.disabled".

**Reply by jattie-ire** [reply]
Thanks
